eng/pipelines/runtime.yml

```
@@ -19,6 +19,10 @@ trigger:
     - LICENSE.TXT
     - PATENTS.TXT
     - THIRD-PARTY-NOTICES.TXT
+    - eng/pipelines/coreclr/perf*yml
+    - eng/pipelines/coreclr/templates/build-perf*
+    - eng/pipelines/coreclr/templates/perf-job.yml
+    - eng/pipelines/coreclr/templates/run-perf*
 
```

Same as above: I think we can also include the below path in the ignored paths, it contains some of the .proj files and the performance-setup files.
```suggestion
    - eng/testing/performance/*
    
```